ansible/roles/build-ossec-deb-pkg/tasks/main.yml

```diff
@@ -1,4 +1,27 @@
 ---
+# Really this is a check to see if we have to create the repo folder,
+# if thats the case we are not under a local vagrant box with a synced
+# root folder.
+- name: Create src path folder
+  file:
+    state: directory
+    path: "{{ item }}"
+  with_items:
+    - "{{ repo_src_path }}"
+    - "{{ build_path }}"
+  register: ossec_src_path_results
+
+- name: Conditional copy over current directory for repo build
+  synchronize:
+    src: ../../../../
+    dest: "{{ repo_src_path }}"
+    use_ssh_args: yes
+    recursive: yes
+    rsync_opts:
+      - "--exclude=*.git"
+      - "--exclude=*.deb"
+  when: ossec_src_path_results.changed
```

Comment:
A few concerns about this `synchronize` task:

1. `synchronize` should be idempotent, so we don't need to add a conditional to it.
2. The conditional as written means that if _local_ files change, they will not make it to the remote host, simply because the target directory already exists.
3. Can't find docs for the `use_ssh_args` parameter; I don't think that's a valid option.
4. `recursive: yes` is the default behavior.
5. We probably also want `delete: yes` to clean up the remote directory.

Will confirm the above observations prior to changing.